src/pages/Settings/Settings.js

```diff
@@ -130,12 +131,13 @@ export function Settings ({ history: { push } }) {
         bottomStyle
         dataTestId='network-type'
         value={!isEmpty(status) && status.networkId ? presentHash(status.networkId, 14) : 'Not Available'} />
-      <div styleName='settings-header'>Access Port Numbers</div>
+      {/* NB: Intentionally commented out for now. Defer to Holo Central. */}
```

Comment:
let's just remove this. Keeping the cancel stuff makes sense because thats waiting on 1 specific bug fix. Who knows when/if we'll ever need this ports stuff.